### PR TITLE
dts: boards: stm32wb5mmg_dk: add buttons support (#73414)

### PR DESCRIPTION
**Pull request by QuentinCaldeira-eaton**
Add support for the two user buttons on the board, as discussed in https://github.com/zephyrproject-rtos/zephyr/issues/73414

### CODE REVIEW

**Comment by erwango** [comment]
Thanks for this contribution, please squash your commits (see contribution guidelines for more explanation) and fix compliance issues reported by Zephyr CI.

File: boards/st/stm32wb5mm_dk/doc/stm32wb5mm_dk.rst [hunk 1]
```diff
@@ -178,6 +178,10 @@ simultaneously. The selection is done by JP4 and JP5 jumpers.
 To use the RGB LED, JP5 must be ON and JP4 OFF. In this configuration,
 GPIO_SELECT2 (PH1) is the chip select for this RGB device on SPI1.
 
+Buttons
+----
+STM32WB5MM-DK has two user buttons. The first button is mapped to PC12,
+and the second to PC13. They are respectively redirected to sw0 and sw1 in zephyr.
```
Comment (ajarmouni-st) [inline boards/st/stm32wb5mm_dk/doc/stm32wb5mm_dk.rst:184]: suggestion
```
- They are respectively redirected to sw0 and sw1 in zephyr.
+ They have the aliases sw0 and sw1 respectively.
```

Comment (QuentinCaldeira-eaton) [reply boards/st/stm32wb5mm_dk/doc/stm32wb5mm_dk.rst]: It's done normally

**Review by ajarmouni-st — APPROVED** [review]

**Review by erwango — CHANGES_REQUESTED** [review]
Thanks for the change, but please fix compliance checks reported below.

**Comment by QuentinCaldeira-eaton** [comment]
Is it because my first and last name are inverted in the commit message ?

**Comment by ajarmouni-st** [comment]
> Is it because my first and last name are inverted in the commit message ?

Yes, can be fixed with `git commit --amend --author="Caldeira Quentin <QuentinCaldeira@eaton.com>"`

**Pull request closed without merging**